rpcs3/Emu/CPU/Backends/AArch64/AArch64Signal.cpp

```diff
@@ -0,0 +1,83 @@
+#include <stdafx.h>
+#include "AArch64Signal.h"
```

Comment:
I feel like all of these files should at least be added to the VS filters

Reply:
VS will try to build them though.

Reply:
I don't think it will if you only add the filters and not the compile sections.
I can add it later